domains/transfer/Transfer.API/API/Claiming/Api/Controllers/ClaimController.cs

```diff
@@ -0,0 +1,108 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using API.Claiming.Api.Dto.Response;
+using API.Claiming.Api.Models;
+using API.Claiming.Api.Repositories;
+using API.Shared.Extensions;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace API.Claiming.Api.Controllers;
+
+[Authorize]
+[ApiController]
+[Route("api/claims")]
```

Comment:
We have decided to not a "root" entity in our API paths. My suggestion is to remove this and then have e.g. 

`GET /api/claim-process`
`POST /api/claim-process/start`
`DELETE /api/claim-process/stop` or `POST /api/claim-process/stop` or `DELETE /api/claim-process`

Reply:
out of curiosity didnt we decide to remove the topic from the guldelines and let the teams decide?

Reply:
fixed eb582eb